src/PresentationalComponents/SystemsTable/SystemsTable.js

```diff
@@ -158,38 +144,35 @@ const SystemsTable = ({ systemsFetchStatus, fetchSystems, systems, intl, filters
         onDelete: () => setSearchText('')
     };
 
+    const handleRefresh = (options) => {
+        setFilters({
+            ...filters,
+            limit: options.per_page,
+            offset: (options.page * options.per_page) -  options.per_page
+        });
+    };
+
     return <React.Fragment>
-        <PrimaryToolbar
-            pagination={{
-                itemCount: results,
-                page: filters.offset / filters.limit + 1,
-                perPage: Number(filters.limit),
-                onSetPage(event, page) { onSetPage(page); },
-                onPerPageSelect(event, perPage) { setFilters({ ...filters, limit: perPage, offset: 0 }); },
-                isCompact: false
-            }}
-            filterConfig={{ items: filterConfigItems }}
-            activeFiltersConfig={activeFiltersConfig}
-        />
-        {systemsFetchStatus === 'fulfilled' &&
-            <Table aria-label={'rule-table'} sortBy={sortBy} onSort={onSort} cells={cols} rows={rows}>
-                <TableHeader />
-                <TableBody />
-            </Table>
+        {
+            InventoryTable && systemsFetchStatus !== 'failed' ?
```

Comment:
Wondering if we shouldn't leave the loading state for the bridge landing on the page and loading the inventory table, presently s just kinda white space 🤔

Reply:
Yeah, good point! Updated.